scripts/commands/getstats.lua

```diff
@@ -41,8 +41,10 @@ commandObj.onTrigger = function(player)
     player:printToPlayer(string.format('Food Accuracy%% bonus: %i ', target:getMod(xi.mod.FOOD_ACCP)), xi.msg.channel.SYSTEM_3)
     player:printToPlayer(string.format('Accuracy Base: %i ', target:getMod(xi.mod.ACC)), xi.msg.channel.SYSTEM_3)
     player:printToPlayer(string.format('Total Accuracy: %i ', target:getStat(xi.mod.ACC)), xi.msg.channel.SYSTEM_3)
+    player:printToPlayer(string.format('Main Weap Dmg: %i ', target:getWeaponDmg()), xi.msg.channel.SYSTEM_3)
```

Comment:
I'd put mainhand wdmg and ranged wdmg together and up at the top eitehr right before or right after the base stats of str/dex/etc

But really, this is getting so large we should start breaking groups of stats into subcommands soon. That was my intent to do when I originally slapped together this series of prints, which I expect is going to get redone from `string.format()` into `fmt` somewhere down the road.